src/main/java/com/cburch/logisim/std/memory/ShiftRegisterHDLGeneratorFactory.java

```diff
@@ -88,11 +88,11 @@ public void getGenerationTimeWiresPorts(Netlist theNetlist, AttributeSet attrs)
         if (nrOfBits == 1) {
           if (Hdl.isVhdl()) {
             for (var stage = 0; stage < nrOfStages; stage++)
-              map.putAll(GetNetMap(String.format("D(%d)", stage), true, comp, 6 + 2 * stage, nets));
+              map.putAll(Hdl.getNetMap(String.format("D(%d)", stage), true, comp, 6 + 2 * stage, nets));
```

Comment:
I'd put multiplication in `()` (same in line below)

Reply:
done